### PR TITLE
Add new hate crimes button, logic to show it, and translations.

### PR DESCRIPTION
**Pull request by billyhunt**
[Link to ZenHub issue.](https://github.com/usdoj-crt/crt-portal-management/issues/1181)

## What does this change?

The hate crimes section of the landing page should link to our internal "hate-crime-human-trafficking.html" page, with translations added for the button in all languages.  

## Screenshots (for front-end PR):

Screen Shot 2022-02-08 at 2 03 01 PM

## Checklist:

+ [x] Refresh your cache.  Then go to the home page, click "Workplace discrimination or other employment-related problem".  From the dropdown select "Victim of a hate crime"
+ [x] is the "Get help for hate crimes" button there?  
+ [x] Click the button and make sure you are taken to the "hate-crime-human-trafficking" page.

**Check the following languages**

+ [x] Spanish
Consiga ayuda en relación con un delito de odio
+ [x] Traditional Chinese
為尋求仇恨犯罪的幫助
+ [x] Simplified Chinese
为寻求仇恨犯罪的帮助
+ [x] Vietnamese
Nhận sự giúp đỡ khi gặp phải những hành vi phạm tội do thù hận
+ [x] Tagalog
Humingi ng tulong para sa mga krimen ng pagka-poot
+ [x] Korean
증오 범죄에 대한 도움 요청

### Author

+ [x] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.).
+ [x] Check for, document, and establish a testing plan for any behavior that may vary across environments or is otherwise difficult to test.
+ [x] Check for [accessibility](https://github.com/usdoj-crt/crt-portal/blob/develop/docs/a11y_plan.md).
+ [x] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

### Reviewer

+ [x] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.).
+ [x] Check for any behavior that may vary across environments or is difficult to test, and ensure that it is well-understood, documented, and that there is a testing plan in place.
+ [x] Re-check for [accessibility](https://github.com/usdoj-crt/crt-portal/blob/develop/docs/a11y_plan.md).
+ [x] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

## Notes for reviewer:

See [PR instructions doc](https://github.com/usdoj/crt-portal/blob/master/docs/pull_requests.md) for full pull request review instructions.

### CODE REVIEW

**Comment by louh** [comment]
The font styling on the two buttons are different. I can see that the button is true to the design mockup, but should they also be consistent in some way?
Screen Shot 2022-02-08 at 3 15 23 PM
Screen Shot 2022-02-08 at 3 15 31 PM

File: crt_portal/cts_forms/templates/landing.html [hunk 1]
```diff
@@ -226,6 +226,9 @@ <h2 class="h2__display">{% trans "Choose from this list to see example civil rig
                     <p>{{ example|safe }}</p>
                   {% endfor %}
                 </div>
+                <div hidden id="crt-landing--hate-crimes">
+                  <a class="usa-button usa-button--outline" aria-label="{% trans 'Get help for hate crimes' %}" href="./hate-crime-human-trafficking">{% trans "Get help for hate crimes" %}</a>
```
Comment (louh) [inline crt_portal/cts_forms/templates/landing.html:230]: I don't think we need to have an  [`aria-label`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label) here that duplicates the link text content.

**Review by louh — CHANGES_REQUESTED** [review]

**Comment by billyhunt** [comment]
Updates.  What do you all think @louh and @chinelo-ikejimba?

Screen Shot 2022-02-10 at 11 00 57 AM
Screen Shot 2022-02-10 at 11 00 50 AM

**Comment by chinelo-ikejimba** [comment]
> Updates. What do you all think @louh and @chinelo-ikejimba?
> 
> Screen Shot 2022-02-10 at 11 00 57 AM
> 
> Screen Shot 2022-02-10 at 11 00 50 AM

@billyhunt great progress! Please un-italicize the hyperlinks and they'll be good to go.

**Review by louh — APPROVED** [review]

**Pull request merged**